### PR TITLE
SCF-Assignment-6

### PR DESCRIPTION
**Pull request by meta-arjita-marelay**
ADT and CDT

### CODE REVIEW

File: SCF-Assignment-6/Question2/PolyTest.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
+package Question2;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class PolyTest {
+
```
Comment (meta-arpit-pittie) [inline SCF-Assignment-6/Question2/PolyTest.java:8]: Create test cases for Corner cases as well

**Review by meta-arpit-pittie — APPROVED** [review]
Add test methods for Corner Cases
